pkg/controller/deployment/deployment_overrides_validation.go

```diff
@@ -32,6 +32,13 @@ func withContainerArgsValidateHook(certmanagerinformer certmanagerinformer.CertM
 		"--v", "-V",
 		// The host and port that the metrics endpoint should listen on. (default "0.0.0.0:9402")
 		"--metrics-listen-address",
+		// Whether an issuer may make use of ambient credentials.
+		// 'Ambient Credentials' are credentials drawn from the environment, metadata services,
+		// or local files which are not explicitly configured in the Issuer API object.
+		// When this flag is enabled, the following sources for credentials are also used:
+		// AWS - All sources the Go SDK defaults to,
+		// notably including any EC2 IAM roles available via instance metadata.
```

Comment:
Add note on specific requirement for GCP.

Reply:
Done